tests/ComputeRegistrarConstant.targets

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+	<Target Name="ComputeRegistrarConstant" BeforeTargets="BeforeBuild">
+		<PropertyGroup Condition="'$(_PlatformName)' == 'iOS' Or $(TargetFramework.EndsWith('-ios')) Or '$(_PlatformName)' == 'tvOS' Or $(TargetFramework.EndsWith('-tvos'))">
+			<IsDynamicRegistrar Condition="'$(ComputedPlatform)' == 'iPhoneSimulator' And '$(Registrar)' == ''">true</IsDynamicRegistrar>
+			<IsDynamicRegistrar Condition="'$(Registrar)' == 'dynamic'">true</IsDynamicRegistrar>
+		</PropertyGroup>
+
+		<PropertyGroup Condition="'$(_PlatformName)' == 'macOS' Or $(TargetFramework.EndsWith('-macos')) Or '$(_PlatformName)' == 'MacCatalyst' Or $(TargetFramework.EndsWith('-maccatalyst'))">
+			<IsDynamicRegistrar Condition="'$(Configuration)' == 'Debug'">true</IsDynamicRegistrar>
```

Comment:
Should this be set only if not already set?

Reply:
This is just a local variable really, and only for tests, so it doesn't matter really.

Reply:
Just noticed it was a test targets :D